### PR TITLE
Fix #2932: Modernize shields copy

### LINKED ISSUE #2932: Update copy in settings & shield stats
| old | new | where |
| --- | --- | --- |
| Block Ads & Tracking | Block cross-site trackers | Settings, Shields (Advanced) |
| HTTPS Everywhere | Upgrade connections to HTTPS | Settings, Shields (Advanced) |
| Ads & Trackers blocked | Trackers & ads blocked | NTP Stats | 
| Fingerprinting Protection | Block fingerprinting | Shields (Advanced) |

### PR DESCRIPTION
**Pull request by kylehickinson**
## Summary of Changes

This pull request fixes #2932

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — CHANGES_REQUESTED** [review]
Update per https://github.com/brave/brave-ios/issues/2932#issuecomment-712412688

**Comment by kylehickinson** [comment]
No update needed code side, Data Saved is tracked separately and will be added in a future version, the other 2 were already being used correctly

File: BraveShared/BraveStrings.swift [hunk 1]
```diff
@@ -446,7 +446,7 @@ extension Strings {
     public static let showBookmarkButtonInTopToolbar = NSLocalizedString("ShowBookmarkButtonInTopToolbar", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Show Bookmarks Shortcut", comment: "Setting to show a bookmark button on the top level menu that triggers a panel of the user's bookmarks.")
     public static let alwaysRequestDesktopSite = NSLocalizedString("AlwaysRequestDesktopSite", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Always Request Desktop Site", comment: "Setting to always request the desktop version of a website.")
     public static let shieldsAdStats = NSLocalizedString("AdsrBlocked", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Ads \nBlocked", comment: "Shields Ads Stat")
-    public static let shieldsAdAndTrackerStats = NSLocalizedString("AdsAndTrackersrBlocked", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Ads & Trackers \nBlocked", comment: "Shields Ads Stat")
+    public static let shieldsAdAndTrackerStats = NSLocalizedString("AdsAndTrackersrBlocked", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Trackers & ads blocked", comment: "Shields Ads Stat")
```
Comment (iccub) [inline BraveShared/BraveStrings.swift:449]: capitalize `Ads` and `Trackers`

rest is good

**Review by iccub — CHANGES_REQUESTED** [review]

File: BraveShared/BraveStrings.swift (hunk 1 above)
Comment (iccub) [reply BraveShared/BraveStrings.swift:449]: sorry, `Ads` only :) 'Trackers' is correct

Comment (kylehickinson) [reply BraveShared/BraveStrings.swift:449]: As far as I am aware, this is how they want it (first word capitalized). @karenkliu can confirm

**Review by iccub — APPROVED** [review]

**Pull request merged**